seriate.py

```diff
@@ -11,8 +11,10 @@
 ortools_version = Version(ortools.__version__)
 ortools6 = Version("6.0.0") <= ortools_version < Version("7")
 ortools7 = Version("7.0.0") <= ortools_version < Version("8")
-if not ortools6 and not ortools7:
-    raise ImportError("No valid version of ortools installed. Please install ortools 6 or 7.")
+ortools8 = Version("8.0.0") <= ortools_version < Version("9")
```

Comment:
I see the problem.

How about changing `ortools7` to `ortools7_or_later` then?
```
ortools7_or_later = ortools_version >= Version("7.0.0")
```

Reply:
Yes, It is good.